### ISSUE TITLE
Netlify-CMS markdown preview not commonmark compliant

### ISSUE THREAD

**Issue by rriemann** [netlify/netlify-cms#548]
**- Do you want to request a *feature* or report a *bug*?**

**- What is the current behavior?**

A long markdown link is not rendered correctly in the preview

**- If the current behavior is a bug, please provide the steps to reproduce.**

Please post this in the body:

```
If you wish to host the MEUSym 2018, please [apply here](https://forum.my-website.org/form?groupname=BETA-CP&title=Call+for+MEU+Symposium+2018+Host&body=%0D%0A%0D%0A%2APlease+conclude+your+message+with+answers+to+the+following+questions%21%2A%0D%0A%0D%0AName+of+your+association%3A%0D%0A%0D%0AName+and+function+of+the+representative%3A%0D%0A%0D%0AEmail+address%3A%0D%0A%0D%0AProposed+city+to+host+the+MEU+Symposium+2018%3A%0D%0A%0D%0AHas+a+member+of+your+association+attended+any+edition+of+the+MEU+Symposium+2015-2017%3F%0D%0A%0D%0A%0D%0ASince+when+has+your+association+been+organising+their+MEU+conference%3F%0D%0A%0D%0AHow+much+manpower+can+you+afford+for+the+MEU+Symposium+in+the+Summer+of+2018%3F+Are+the+people+from+your+MEU+team%3F%0D%0A%0D%0AUsually+the+Symposium+is+planned+for+July+-+is+it+a+suitable+date+for+you%3F+If+not%2C+which+month%3F%0D%0A%0D%0ADo+you+have+proposals+or+ideas+for+the+accommodation+during+the+event%3F+If+possible%2C+include+an+estimation+of+costs.+%2840-50+participants%29%3A%0D%0A%0D%0ADo+you+have+access+to+university+rooms+or+other+venues+free+of+charge+for+NGOs%3F+If+not%2C+what+venues+would+you+recommend%3F%0D%0A%0D%0ACan+you+give+us+a+rough+estimate+on+the+total+cost+of+the+event%2C+counting+for+50+people%3F+%28please%2C+remember%2C+in+the+E%2B+framework+staff+is+not+paid%29%0D%0A%0D%0AAnd+lastly%3A+what+makes+your+city+unique%3F%0D%0A) to reach out to the BETA Europe Concepts and Partnerships [team](https://forum.beta-europe.org/groups/BETA-CP). You may need to signup first. Alternatively, you can direct your application to the [BETA office](mailto:mymail@my-website.org?subject=Call%20for%20MEU%20Symposium%202018%20Host&body=%0A%0A*Please%20conclude%20your%20message%20with%20answers%20to%20the%20following%20questions!*%0A%0AName%20of%20your%20association%3A%0A%0AName%20and%20function%20of%20the%20representative%3A%0A%0AEmail%20address%3A%0A%0AProposed%20city%20to%20host%20the%20MEU%20Symposium%202018%3A%0A%0AHas%20a%20member%20of%20your%20association%20attended%20any%20edition%20of%20the%20MEU%20Symposium%202015-2017%3F%0A%0A%0ASince%20when%20has%20your%20association%20been%20organising%20their%20MEU%20conference%3F%0A%0AHow%20much%20manpower%20can%20you%20afford%20for%20the%20MEU%20Symposium%20in%20the%20Summer%20of%202018%3F%20Are%20the%20people%20from%20your%20MEU%20team%3F%0A%0AUsually%20the%20Symposium%20is%20planned%20for%20July%20-%20is%20it%20a%20suitable%20date%20for%20you%3F%20If%20not%2C%20which%20month%3F%0A%0ADo%20you%20have%20proposals%20or%20ideas%20for%20the%20accommodation%20during%20the%20event%3F%20If%20possible%2C%20include%20an%20estimation%20of%20costs.%20(40-50%20participants)%3A%0A%0ADo%20you%20have%20access%20to%20university%20rooms%20or%20other%20venues%20free%20of%20charge%20for%20NGOs%3F%20If%20not%2C%20what%20venues%20would%20you%20recommend%3F%0A%0ACan%20you%20give%20us%20a%20rough%20estimate%20on%20the%20total%20cost%20of%20the%20event%2C%20counting%20for%2050%20people%3F%20(please%2C%20remember%2C%20in%20the%20E%2B%20framework%20staff%20is%20not%20paid)%0A%0AAnd%20lastly%3A%20what%20makes%20your%20city%20unique%3F) (per mail with this link).
```

**- What is the expected behavior?**

That is works as on http://spec.commonmark.org/dingus/ and the link is rendered correctly

**Comment by erquhart** [comment]
This is addressed in #254 - test here: https://deploy-preview-254--cms-demo.netlify.com/#/collections/posts/entries/new

**Comment by erquhart** [comment]
Fixed in #254, closing.
